packages/dataviews/src/types.ts

```diff
@@ -368,7 +368,9 @@ export interface ActionButton< Item extends AnyItem >
 	/**
 	 * The callback to execute when the action is triggered.
 	 */
-	callback: ( items: Item[] ) => void;
+	callback: (
+		items: Item[]
+	) => void | ( ( { registry }: { registry: any } ) => void );
```

Comment:
Unfortunately, we can't be more specific for the registry type just yet because the data package is not fully typed yet.